client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -39,6 +39,7 @@ import {
 	isRenewable,
 	isWithinIntroductoryOfferPeriod,
 	isIntroductoryOfferFreeTrial,
+	hasPaymentMethod,
```

Comment:
Nit: while the existing list isn't alphabetised, can we move `hasPaymentMethod` above all the `is*` imports?

Reply:
Are we missing a prettier rule, or at least a lint rule, here? My cursory scan of the project tells me we don't sort members in most cases; if we should sort new code, then I think that should be automated or at least auto-checked.

Reply:
I checked, and there is an eslint rule: `sort-keys`. 

https://eslint.org/docs/latest/rules/sort-keys

It was also requested in the prettier repo, but they warned against it because it can affect iteration and execution order. 

https://github.com/prettier/prettier/issues/1096

I will just manually change it here.

Reply:
We should be able to get eslint to enforce and auto-fix this:

https://dev.to/julioxavierr/sorting-your-imports-with-eslint-3ped

I will move it manually in this case.

Reply:
This one is less a canonical/authoritative requirement, but more of a personal preference (which I shared with my previous team), mostly because it can make it simpler to scan through the list of imports when they're from the same source, and when working out what values we're destructuring.

So this might be worth a broader discussion with the Calypso team, if only to gauge whether people do (not) align with the preference.